mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/MavenPropertyResolver.java

```diff
@@ -8,8 +8,10 @@
 
     private static final Pattern TEST_CLASS_PATTERN = Pattern.compile("[^a-z0-9 ]", Pattern.CASE_INSENSITIVE);
 
+    private static boolean skipTests = false;
+
     static boolean isSkipTestExecutionSet() {
-        return isSkipTests() || isSkip();
+        return isSkipTests() || isSkip() || skipTests;
```

Comment:
having `isSkipTests()` method and `skipTests` variable with `setSkipTests` method that take care of something different than the first one is pretty confusing